### PR TITLE
Add new Discourse 7.0.0 chart major update notes

### PR DESCRIPTION
**Pull request by marcosbc**
Signed-off-by: Marcos Bjoerkelund <marcos@bitnami.com>

### CODE REVIEW

File: charts/discourse/_upgrade.md.erb [hunk 1]
```diff
@@ -9,6 +9,24 @@ weight: 20
 highlight: 20
 =end %>
 
+### To 7.0.0
+
+This major upgrades the Discourse version to 2.8.0. This version includes a breaking change in the *lazy-yt* plugin, and the recommendation is to remove it, or manually upgrade it. To do so, follow these steps:
+
+1. Upgrade to the latest version of the *bitnami/discourse* chart with Diagnostics mode:
```
Comment (juan131) [inline charts/discourse/_upgrade.md.erb:16]:
```suggestion
This major upgrades the Discourse version to *2.8.0*.

#### What changes were introduced in this major version?

This version includes a breaking change in the *lazy-yt* plugin, and the recommendation is to remove it, or manually upgrade it.

#### Upgrading Instructions

To upgrade to *7.0.0* from *6.x*, follow these steps below:
```

**Review by juan131 — CHANGES_REQUESTED** [review]
Please follow the "Upgrade notes format" we used in the past.

**Review by juan131 — APPROVED** [review]
LGTM

**Review by migruiz4 — APPROVED** [review]
LGTM, but please fix DCO

**Pull request merged**